qa-covid19.planx-pla.net/manifest.json

```diff
@@ -4,33 +4,33 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.04",
-    "augur": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-augur:fix_docker2",
-    "awshelper": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/awshelper:2021.04",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.07",
+    "augur": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-augur:2021.07",
```

Comment:
i think we don't even need augur in the manifest... could you check with @thanh-nguyen-dang, and remove it in QA and prod if we don't need it?
if we need it, we need a create a 0.1.0 tag and deploy that. There is no 2021.07.

Reply:
checked with Thanh / Yilin if we need augur in the manifest.json. @yilinxu mentioned it is ok to remove augur from the manifest. I will remove it from PROD manifest when deploying the new versions to it.

Reply:
nice, let's remove it here too then 👍